### PR TITLE
Board

### PR DESCRIPTION
**Pull request by MauricioRobayo**

### CODE REVIEW

File: bin/main.rb [hunk 1]
```diff
@@ -1,92 +1,28 @@
-require_relative '../lib/game_layout.rb'
 require_relative '../lib/game.rb'
 
-game = Game.new
+puts <<-'TITLE'
```
Comment (Ispirett) [inline bin/main.rb]: All your logic should be in classes.

File: bin/main.rb [hunk 2]
```diff
@@ -1,92 +1,28 @@
...
+       _______\/\\\_______\/\\\__\///\\\\\\\\_____________________\/\\\_______\//\\\\\\\\/\\__\///\\\\\\\\_____________________\/\\\________\///\\\\\/____\//\\\\\\\\\\___________  
+        _______\///________\///_____\////////______________________\///_________\////////\//_____\////////______________________\///___________\/////_______\//////////____________ 
 
 
+TITLE
 
-win = false
 
 
+print "Player 1 name: "
```
Comment (Ispirett) [inline bin/main.rb]: This logic also should be in a class.

File: lib/game.rb [hunk 3]
```diff
@@ -1,54 +1,37 @@
...
-  end
 
-  def check_consecutive?(orientation)
-    if orientation.all? { |cell| cell == 'X' }
-      @winner = 'X'
-      return true
-    elsif orientation.all? { |cell| cell == 'O' }
-      @winner = 'O'
-      return true
+    if @board.winner?
```
Comment (Ispirett) [inline lib/game.rb:32]: This logic should be inside of a method

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey @MauricioRobayo @nicolasmarino99   Excellent Job with this Project love the graphics lol :100: 
image. 

**There a just some minor changes to be made.**

File: lib/game.rb (hunk 3 above)
Comment (nicolasmarino99) [reply lib/game.rb:32]: HI @Ispirett, thanks for your code review, however we didn´t understand quite well this last point from the review. Here you have our [game file](https://github.com/nicolasmarino99/Tic-tac-toe-Ruby/blob/board/lib/game.rb). Thanks again for you review. Have a great day!

**Review by Ispirett — APPROVED** [review]
Hey @MauricioRobayo @nicolasmarino99 Excellent work guys :1st_place_medal: 

image

Looks good works good is good :+1:  You can merge now 
good  luck on your next project

**Pull request merged**